src/server/routes/consents.ts

```diff
@@ -347,6 +349,10 @@ router.post(
       const { update, pageTitle: _pageTitle } = consentPages[pageIndex];
       pageTitle = _pageTitle;
 
+      // If on the first page, attempt to update location for consented users.
+      pageTitle == 'Stay in touch' &&
+        updateRegistrationLocationViaIDAPI(req.ip, sc_gu_u, req);
+
```

Comment:
I'd suggest looking at the `pageIndex` here, and using an `if` statement for readability, also using the strict equality (`===`)

```suggestion
      // If on the first page, attempt to update location for consented users.
      if (pageIndex === 0) {
        updateRegistrationLocationViaIDAPI(req.ip, sc_gu_u, req);
      }
```

If you really wanted to use `pageTitle` for some reason then i suggest using `CONSENTS_PAGES.CONTACT` instead

```suggestion
      // If on the first page, attempt to update location for consented users.
      if (pageTitle === CONSENTS_PAGES.CONTACT) {
        updateRegistrationLocationViaIDAPI(req.ip, sc_gu_u, req);
      }
```

Reply:
Very obvious suggestion 😁👍